packages/terra-pills/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "terra-pills",
```

Because this package is scoped this should be `@cerner/terra-pills`

```suggestion
  "name": "@cerner/terra-pills",
```